plugin/trino-hudi/pom.xml

```diff
@@ -15,7 +15,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.hudi.version>0.11.1</dep.hudi.version>
+        <dep.hudi.version>0.12.1</dep.hudi.version>
```

Comment:
With this, I think we can also remove the `--add-opens` configs: https://github.com/trinodb/trino/blob/master/plugin/trino-hudi/pom.xml#L461-L467

Reply:
Thanks for this information.